core/src/main/java/io/ultrabrew/metrics/reporters/SLF4JReporter.java

```diff
@@ -32,6 +37,10 @@
  */
 public class SLF4JReporter extends TimeWindowReporter {
```

Comment:
Every change on this class looks like it could be in a completely separate PR, any reason why thats not the case?

Reply:
This change is to demonstrate how the reporters are going to use the HistogramAggregator.

Reply:
Hmmh, can't see it anywhere? Usage of HistogramAggregator is not in default aggregators or anywhere?

Reply:
You can see the usage in SLF4JReporterTest